web/src/gear_menu.ts

```diff
@@ -127,14 +129,22 @@ export function initialize() {
                             blueslip.warn(err);
                             return;
                         }
-                        if (r.status !== "OK") {
-                            blueslip.warn(r);
+
+                        const webathena_response_schema = z.object({
+                            status: z.string(),
+                            session: z.unknown(),
+                        });
+
+                        const res = webathena_response_schema.parse(r);
+
+                        if (res.status !== "OK") {
+                            blueslip.warn(JSON.stringify(res));
```

Comment:
A response with a `status` other than `"OK"` will not have a `session`, and will crash on `.parse` before we get here. We need to use a more accurate schema and/or `.safeParse`.

Context: https://github.com/davidben/webathena/blob/master/app/scripts-src/request_ticket.js